backend/tests/models/test_osrm.py

```diff
@@ -20,9 +20,10 @@ def setup(self):
 
     def test_measure_between_two_points(self):
         """Check that the OSRM time matches expectations."""
-        d = self.measurer.measure_between_two_points(NEWPORT_RI, CLEVELAND_OH)
+        measurements = self.measurer.measure_one_to_many_points(NEWPORT_RI, [CLEVELAND_OH])
+        # measure_one_to_many_points return measurements in seconds.
         # The time should be more than 10 hours (600 minutes) but less than 20 hours (1200 minutes).
-        assert abs(d - 900.0) < 300.0
+        assert abs(float(measurements[0]) / 60 - 900.0) < 300.0
```

Comment:
So `measure_one_to_many_points` returns times in seconds, but closest returns times in minutes?

Reply:
Yeah, not sure where the best place is to make the change